.github/workflows/test-build-deb.yml

```diff
@@ -271,14 +281,24 @@ jobs:
         java: [14]
     steps:
       - uses: actions/checkout@v1
+
+      - name: Getting OD Version
+        run: |
+          cd elasticsearch/bin
+          OD_VERSION=`./version-info --od`
+          plugin_tag=v$OD_VERSION.0
+          echo "::set-env name=tag::$plugin_tag"
+
```

Comment:
Why not set a global env, I dont think you need to get the value multiple times.

Reply:
Scope of env is limited to a job. Any idea how to create a global one?

Reply:
https://github.com/opendistro-for-elasticsearch/opendistro-build/blob/master/.github/workflows/check_plugin.yml#L10

I think this works, you first define the global env, then use the first check version block to update it, then it should work across multiple jobs. If not, then I will just accept this as a temp solution.

Reply:
i don't think it will work.  the workflow execution is running at job level. the jobs can run in parallel in multiple run containers. It doesn't make sense to propagate this update to other jobs

Reply:
Well seems like we can share the value/data across jobs in a different way, just not within env variables. However, this seems not anyway better for our purpose.
https://help.github.com/en/actions/configuring-and-managing-workflows/persisting-workflow-data-using-artifacts

Will accept the current implementation until we have a better solution.

@gaiksaya @weicongs-amazon

Reply:
We can modify it before the job starts, like in this case the testing plugins job depends on "sign-deb-artifacts". So in that job we can set the env variable.